docs/Makefile

```diff
@@ -1,50 +1,71 @@
+CURRENT_DIR = $(shell pwd)
```

Comment:
Nit: this can be replaced with `$(CURDIR)`

Reply:
Oh nice, thats good to know. I'll update that.